lib/mutant/isolation.rb

```diff
@@ -1,31 +1,35 @@
 module Mutant
   # Module providing isolation
-  module Isolation
+  class Isolation
     Error = Class.new(RuntimeError)
 
-    module None
-
+    class None < self
       # Call block in isolation
       #
       # @return [Object]
       #
       # @raise [Error]
       #   if block terminates abnormal
-      def self.call(&block)
+      def call(&block)
         block.call
```

Comment:
True, I think this is the better pattern. I think I mentally prefer `&block` because it advertises "this method takes a block" something ruby should have enforced in the first place.

But I cannot / should not compensate this via redundant syntax.